### PR TITLE
{numlib}[foss/2022a] Trilinos v13.4.1 w/ Python 3.10.4

### PR DESCRIPTION
**Pull request by branfosj**
(created using `eb --new-pr`)

* [x] #17447
* [x] https://github.com/easybuilders/easybuild-easyblocks/pull/2900

This is based on a build I've done locally of 13.2.1. This is not the fuller build we've had for 12.x versions.

### CODE REVIEW

**Comment by branfosj** [comment]
Test report by @branfosj
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/2900
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0104u07a.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/c4ab5698058b6910c76557fa6a57928b for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso
CORE_CNT=16
EB_ARGS="--include-easyblocks-from-pr 2900"

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2
CORE_CNT=16
EB_ARGS="--include-easyblocks-from-pr 2900"

**Comment by boegel** [comment]
Test report by @boegel
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/2900
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in total)
node3118.skitty.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/ead5b463613c4886516e2f6ca71aba59 for a full test report.

File: easybuild/easyconfigs/t/Trilinos/Trilinos-13.4.1-foss-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+homepage = 'https://trilinos.org'
+description = """The Trilinos Project is an effort to develop algorithms and enabling technologies
+ within an object-oriented software framework for the solution of large-scale, complex multi-physics
+ engineering and scientific problems. A unique design feature of Trilinos is its focus on packages."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+toolchainopts = {'usempi': True, 'pic': True, 'strict': True}
+
+source_urls = ['https://github.com/trilinos/Trilinos/archive/refs/tags/']
+sources = ['trilinos-release-13-4-1.tar.gz']
```
Comment (boegel) [inline easybuild/easyconfigs/t/Trilinos/Trilinos-13.4.1-foss-2022a.eb:15]: @branfosj We should probably avoid the hardcoded version here?

```suggestion
sources = ['trilinos-release-%star.gz' % '-'.join(version.split('.'))]

**Review by boegel — CHANGES_REQUESTED** [review]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/2900
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3518.doduo.os - Linux RHEL 8.6, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/67327e4ea7bc7116c0c3571c946a8f7f for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/2900
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/419e0750cd24b359f9e7bef8d07b0121 for a full test report.

**Comment by verdurin** [comment]
@boegel are you happy with this one now?

**Comment by boegel** [comment]
Going in, thanks @branfosj!

**Pull request merged**